api/envoy/extensions/grpc_credentials/file_based_metadata.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+// [#protodoc-title: Grpc Credentials File Based Metadata]
+// Configuration for File Based Metadata Grpc Credentials Plugin
+
+package envoy.extensions.grpc_credentials;
```

Comment:
Also, this should live in the https://github.com/envoyproxy/envoy/tree/master/api/envoy/config subtree.